app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -0,0 +1,23 @@
+module V0
```

Comment:
> That way we're only ever maintaining two versions - the "latest" and whatever is experimental/volatile/early development

Yes. Version numbering is more important for phone apps because old phone clients never totally go away because some people never upgrade, but in our case, depending on how we do deploys, we shouldn't have users more than 1 version behind the times at any given moment.